sdk/include/cardboard.h

```diff
@@ -433,6 +433,29 @@ void CardboardQrCode_getSavedDeviceParams(uint8_t** encoded_device_params,
 ///     using cardboard_device.proto.
 void CardboardQrCode_destroy(const uint8_t* encoded_device_params);
 
+/// Sets and saves the encoded device parameters provided by the uri.
```

Comment:
I think this should be now "Sets...", e.g.: "Saves the encoded device parameters provided by an URI."